catalog/spatial/registry/registry-federation-admin-impl/src/main/java/org/codice/ddf/registry/federationadmin/impl/FederationAdmin.java

```diff
@@ -415,12 +458,80 @@ public boolean registryStatus(String servicePID) {
     }
 
     @Override
+    public Map<String, Object> registryMetacard(String registryId) {
+        if (!RegistryUtility.validRegistryId(registryId)) {
+            throw new IllegalArgumentException("Passed in registryId is invalid. Must conform to "
```

Comment:
I thought about it but if it is a malicious request i didn't want to put that in the error message